scaleway/resource_function_namespace.go

```diff
@@ -0,0 +1,160 @@
+package scaleway
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	function "github.com/scaleway/scaleway-sdk-go/api/function/v1beta1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+func resourceScalewayFunctionNamespace() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceScalewayFunctionNamespaceCreate,
+		ReadContext:   resourceScalewayFunctionNamespaceRead,
+		UpdateContext: resourceScalewayFunctionNamespaceUpdate,
+		DeleteContext: resourceScalewayFunctionNamespaceDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Default: schema.DefaultTimeout(defaultFunctionNamespaceTimeout),
+		},
+		SchemaVersion: 0,
+		Schema: map[string]*schema.Schema{
```

Comment:
What about the `secret_environment_variables` [attribute](https://github.com/scaleway/scaleway-sdk-go/blob/810e5a35c5722efd2b28bd967efaeec4651bd5d0/api/function/v1beta1/function_sdk.go#L798)

Reply:
At the moment the way secret_environment_variables is implemented is a bit weird. I would like to talk a bit with the product team before implementing it.